### PR TITLE
ful_15941_fix varnishcache added to integrations

### PR DESCRIPTION
**Pull request by fulyauluturk**

### CODE REVIEW

File: docs/integration/varnishcache-logs.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+<img
+  class="content-modal-image"
+  alt="Varnish Cache Backend Side Logs Report"
+  src="../../images/integrations/varnishcache-backend-side-logs.png"
+  title="Varnish Cache Backend Side Logs Report"
+/>
+
+## Troubleshooting
+
+If you have trouble sending logs, try out the latest version of [Sematext Agent](../agents/sematext-agent/installation/). Also, make sure Sematext Agent is configured to send logs to your OpenSearch Logs App. Last, check the [Log Agents panel](https://sematext.com/docs/fleet/#log-agents) for any errors, and refer to our [Sematext Logs FAQ](https://sematext.com/docs/logs/faq/) for useful tips.
```
Comment (boratanrikulu) [inline docs/integration/varnishcache-logs.md:87]: `OpenSearch Logs App`

Comment (fulyauluturk) [reply docs/integration/varnishcache-logs.md]: Thanks Bora, fixed.

File: docs/integration/varnishcache-logs.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+## Log Collection
+
+1. To enable Varnish logging uncomment the following in `/etc/default/varnishncsa`:
+
+``` bash
+  VARNISHNCSA_ENABLED=1
+```
+
+2. Create `st_varnishncsa_format` file and add the following:
```
Comment (cpipilas) [inline docs/integration/varnishcache-logs.md:14]: where the file should be created?

Comment (fulyauluturk) [reply docs/integration/varnishcache-logs.md]: Under root. Added thanks :)

**Review by cpipilas — APPROVED** [review]

**Pull request merged**